Server/SQL/procedures.sql

```diff
@@ -272,7 +272,8 @@ BEGIN
         L.last_update_device,
         Deck.share_code,
         User.name AS author,
-        L.accession
+        L.accession,
```

Comment:
you have server changes in this PR and the same changes in #156. I would remove them from this pr and use the separate PR